evap/static/scss/_utilities.scss

```diff
@@ -71,3 +71,9 @@ a.no-underline:hover {
 .z-over-fixed {
     z-index: $zindex-fixed + 1;
 }
+
+@for $i from 1 through 100 {
+    .width-percent-#{$i} {
+        width: $i * 1%;
+    }
+}
```

Comment:
Hmm, no strong opinion from my side. Two thoughts:
* For my use cases here, I don't need the `!important`
* It feels weird to re-define bootstrap helper classes -- if they change, we override their definition
* iirc, for grids, lots of setups use "parts of 12" or "parts of 24" to define dimensions. "percent" in the name makes it more clear what "width-percent-5" means, compared to "w-5".

In the end, I don't really care, we can also use `w-X` here. Including 0 is fine with me.

Reply:
- I think `!important` is appropriate, because we never want the width to be replaced by a more specific rule. I am 80% sure such a problem could come up naturally and think we cannot go wrong with adding `!important`, so I would prefer adding it.
- Fair enough, I think it is unlikely to happen though
- I am not aware of the "of 12" rule, but I don't know too much about bootstrap. It's a tradeoff between initial clarity and familiarity with a known pattern.

To be honest, I thought that we used more of `w-xx` in evap than a code search actually turned up, so I now also don't care that much anymore. Either way is fine, maybe `width-percent-xx` is actually nicer going forward